VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -32,15 +32,17 @@ private DiscussionTextUtils() {
 
     public static void setAuthorAttributionText(@NonNull TextView textView,
                                                 @StringRes final int stringRes,
```

Comment:
Might be a little cleaner to break this into two parts: One that just generates the string and the other that applies it.